setup.py

```diff
@@ -1,11 +1,31 @@
 from setuptools import setup, Extension
 import os, platform, sys, re
 
-# look for environment variable that specifies path to SCIP Opt lib and headers
+# look for environment variable that specifies path to SCIP
 scipoptdir = os.environ.get('SCIPOPTDIR', '').strip('"')
-includedir = os.path.abspath(os.path.join(scipoptdir, 'include'))
+
+# now check whether SCIP is installed on the system or whether a source code directory has been specified
+if os.path.exists(os.path.join(scipoptdir, 'include')):
+    includedir = os.path.abspath(os.path.join(scipoptdir, 'include'))
+else:
```

Comment:
Users should always first install SCIP. Then there is no need to search for a `src` directory.

Reply:
But this is the point: I want to also use by development version that is installed somewhere else.

Reply:
Also development versions can be installed to some local directory. We always require a `lib` and an `include` directory.

Reply:
Yes, but it is easier to just use ``src`` and use SCIP in place.

Reply:
I have changed to code to just determine whether SCIP has been used "in place". Otherwise nothing changes.